### DISCUSSION TITLE
Is it possible to run Jsign application inside a container?

### DISCUSSION THREAD

**Discussion by nauni77** [ebourg/jsign#248] [Q&A]
Hello,

is it possible to put a Jsign application into an image/ container and make it work? 

I hope there are special devices to mount into the container and it works? If yes, can you name the devices to mount?

Thanks in advance and best regards

Oli

**Comment by ebourg** [comment]
Yes Jsign should work in a container. Additional configuration may be required if you sign with a hardware token, but I can't help you on this topic.

**Comment by nauni77** [comment]
That sounds great. Thanks for the information and the help you offered.

Today I will create an image of my java application. Should I take care of something special? Or do I create a usual container and "only" add some mounts to the container runtime later?

**Reply by ebourg** [reply]
Don't forget to install the SafeNet client in the image if you use an eToken
